DnD6/UI/Inventory/Slot/slot.gd

```diff
@@ -0,0 +1,28 @@
+extends PanelContainer
+
+signal slotClicked(index: int, button: int, double: bool)
+
+
+@onready var textureRect := $MarginContainer/TextureRect
+@onready var quantityLabel := $QuantityLabel
+
+#desc Sets the data for this slot
```

Comment:
add "for inventory slot" or explain "this slot"

Reply:
it's in the Slot Script... What Slot do you think it is refrerring to

Reply:
the slot*